ci_framework/roles/edpm_prepare/tasks/main.yml

```diff
@@ -29,6 +29,15 @@
     name: 'install_yamls_makes'
     tasks_from: 'make_crc_storage'
 
+- name: Attach default network to CRC
+  vars:
+    make_crc_attach_default_interface_env:
+      PATH: "{{ cifmw_path }}"
```

Comment:
I think we don't need to pass down the PATH to the `install_yamls_makes` tasks. IIRC @pablintino ensured it's in the `make` environment